.github/workflows/ci-run.yml

```
@@ -421,6 +421,17 @@ jobs:
             - name: Skip Rust jobs for non-Rust change scope
               run: echo "No Rust-impacting files changed. Rust lint/test/build skipped."
 
+    no-linear-product-refs:
+        name: No-Linear Product Reference Guard
+        needs: [changes]
+        runs-on: [self-hosted, Linux, X64, aws-india, light, cpu40]
```

actionlint (1.7.11)

[error] label "light" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows", "blacksmith-2vcpu-ubuntu-2404", "aws-india", "hetzner", "Linux", "X64". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)